boards/nxp/mr_canhubk3/mr_canhubk3.dts

```diff
@@ -23,6 +23,7 @@
 		zephyr,itcm = &itcm;
 		zephyr,dtcm = &dtcm;
 		zephyr,code-partition = &code_partition;
+		zephyr,uart-mcumgr = &lpuart2;
```

Comment:
I haven't tried using mcumgr on `mr_canhubk3` board, so if you haven't either, I'm not sure makes sense to set this chosen now

Reply:
> I haven't tried using mcumgr on `mr_canhubk3` board, so if you haven't either, I'm not sure makes sense to set this chosen now

It makes sense for any board supporting MCUBoot partitions and a serial interface.

Reply:
>It makes sense for any board supporting MCUBoot partitions and a serial interface.

there's no MCUboot partitions defined for this board

Reply:
You are right. Fixed. 
Thank you.